### ISSUE TITLE
Multiple Bugs with PTB 

### ISSUE THREAD

**Issue by Rekulous** [rauenzi/BetterDiscordApp#587]
There are a bunch of bugs I discovered when just installing BandagedBD on DiscordPTB. I tried troubleshooting them with the repair BandagedBD option in the installer, but nothing happened.

The first one is a popup on the top of PTB showing me that I should download the Desktop version of Discord even though I am using an app. 

The second bug is that there is no Minimize, Maximize, or Exit X Button at the top of the Discord Window. 

The Last Bug that I got is that there is no BandagedBD Menu in the settings. I probably cannot even add any plugins or themes!

**To Reproduce**
I honestly don't know how to reproduce this behavior because I just installed PTB with BandagedBD.

**Expected Behavior**
I expected BandagedBD to boot up perfectly fine just like I had before with Discord Stable version. I was going to try out some plugins.

**Screenshots**
First Bug: 
![image](https://user-images.githubusercontent.com/79665689/111083306-4da47c00-84e3-11eb-94b9-c6fb4d9bc376.png)

Third Bug: 
![image](https://user-images.githubusercontent.com/79665689/111083458-1c787b80-84e4-11eb-9821-104e3c7b43f7.png)

**Discord Version**
Discord PTB

Additional Context:
Installed BandagedBD on Windows 10 and had PTB open while installing. 
Re-Installed PTB multiple times to see if it was going to fix the problem, but nothing happened
Installed BandagedBD right after I installed PTB and signed in to my Discord account.

**Comment by Tropix126** [comment]
This is a known issue and there's a pending fix. This also applies to canary injections.

**Comment by Rekulous** [comment]
> This is a known issue and there's a pending fix. This also applies to canary injections.

Alright,  thanks for letting me know!

**Comment by Helhest** [comment]
I had the same problem too, i unistalled discord and re-installed it and that fixed it.

**Comment by Rekulous** [comment]
> I had the same problem too, i unistalled discord and re-installed it and that fixed it.

Didn't work for me

**Comment by Giotje** [comment]
Having this problem right now aswell, gladly there's a pending fix

**Comment by Giotje** [comment]
I have this issue on stable build of discord too

**Comment by Tenchanie** [comment]
I have the same problem. I hope they will fix it soon.

**Comment by Tenchanie** [comment]
So I have the normal discord downloaded from discord.gg. 3 days ago I updated discord and then I had the same issue like you. I couldn't find the solution, BUT I found out that there's no app folder named with the current patch. I think BandagedBD now can't find the folder and maybe it creates one, or idk but it destroyes discord basically. Maybe it was intentional that discord renamed or deletet this file so we can't use BetterDiscord.
